src/ctia/auth/jwt.clj

```diff
@@ -72,6 +71,11 @@
   (get-in-config [:ctia :auth :casebook :scope]
                               "casebook"))
 
+(defn assets-root-scope [get-in-config]
+  (if (= "private-intel" (entity-root-scope get-in-config))
```

Comment:
I think you may probably loose this `if` statement, make it exactly like `casebook-root-scope` but for `asset`